RoosterPlanner.Api/Controllers/PersonsController.cs

```diff
@@ -1,19 +1,19 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Threading.Tasks;
 using AutoMapper;
-using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Graph;
 using RoosterPlanner.Api.Models;
+using RoosterPlanner.Api.Models.Enums;
 using RoosterPlanner.Common;
-using RoosterPlanner.Models;
-using RoosterPlanner.Models.FilterModels;
 using RoosterPlanner.Service;
 using RoosterPlanner.Service.DataModels;
+using Person = RoosterPlanner.Models.Person;
 
 namespace RoosterPlanner.Api.Controllers
 {
+    [Authorize]
     [Route("api/[controller]")]
     [ApiController]
     public class PersonsController : ControllerBase
```

Comment:
Class names zou ik _singular_ houden. In dit geval is het dan een PersonController. Het enige wat ik in plural zou schrijven zijn collections.

Een voorbeeld in plural:
`foreach (person in persons) { ... }`

Dit draagt ook bij aan de leesbaarheid van de code.

Reply:
Eens. Het is aangepast in de volgende commit

Reply:
Ik gebruik ook wel singular namen, maar in het kader van REST-API zou plural beter passen:
api/persons (lijst van personen) / api/persons/{id} (1 specifiek persoon)
Bovenstaande klopt eigenlijk beter dan api/person voor lijst.

Reply:
Aangepast naar plural